save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/NewDemoToolRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.saveourtool.save.demo
+
+/**
+ * @property organizationName name of an organization on GitHub that maintains the tool
+ * @property projectName name of repo on GitHub
+ * @property vcsTagName release tag that defines the version to be fetched
+ */
+data class NewDemoToolRequest(
```

Comment:
This seems to be very connected with yours `GithubRepo`, right?

Could we make this classes more closer, rename them in terms `GithubRepo` and `GithubRepoDto`, or I missed something?

Reply:
The point is that `GitRepo` defines the source of the tool while `NewDemoToolRequest` specifies the version also

It could be `ToolDto`, but `Tool` also provides `executableName`...